assets/src/design-system/theme/helpers/mq.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { css } from 'styled-components';
+
+export const mq = {
```

Comment:
This is probably a matter of preference, but why not add the breakpoints themselves to the theme? Avoids having magic numbers in a helper function and keeps definitions in one place.

The helper could just take the value from the theme.

Also, could we name this `mediaQueries`? `mq` doesn't mean much.

---

Could also make it a function

```js
export const mediaQueries = (key) => {
  return style => `@media ${theme.breakpoints[key]} { ${style} }`;
};
```

or even a tagged template literal

Reply:
I like this suggestion.

Reply:
Yea I wanted to do a tagged template literal, but there's no way to get anything else to maintain syntax highlighting in vscode besides the `css` tagged template literal.

I'd be happy to update the name to the unabbreviated version and pull in the breakpoints from the theme.

passing in a key is nasty imo cus then when you want to use this you have to not only pull in the function, but also a constant to identify which breakpoint. Just making it a property alleviates this pain

Reply:
Happy to change it if my preferences are outnumbered lol

Reply:
I just updated it to do media queries like how the existing theme provider sets them up. Works well and I think addresses everyone's pain points mentioned here